melodic.ignored

```diff
@@ -0,0 +1 @@
+ur_kinematics
```

Comment:
Wouldn't this have to be added to the release repository?

Reply:
I was going after https://answers.ros.org/question/66545/to-ignore-some-pkgs-upon-release-by-bloom/?answer=66548#post-id-66548 but you might be right. I've never done that.